src/main/scala/com/monsanto/arch/cloudformation/model/resource/EC2.scala

```diff
@@ -167,22 +168,22 @@ object `AWS::EC2::Route` extends DefaultJsonProtocol {
     def read(json: JsValue) = ???
```

Comment:
Yes, it's more implicit chinning: Some code really wants DefaultJsonProtocols, even though all we ever want to provide is DefaultJonWriters